### PR TITLE
Toolkit concept docs XAML: batch 2

### PR DESCRIPTION
**Pull request by williambohrmann3**
Updates the conceptual docs "Usage" section for the following toolkit components:
- Legend
- OverviewMap
- MeasureToolbar

### CODE REVIEW

**Review by ThadT — APPROVED** [review]
Thanks, @williambohrmann3, LGTM 👍

File: docs/legend.md [hunk 1]
```diff
@@ -13,6 +13,42 @@ Display a legend for a map or scene view.
...
+
+```xml
+<Grid xmlns:esri="using:Esri.ArcGISRuntime.UI.Controls"
+      xmlns:toolkit="using:Esri.ArcGISRuntime.Toolkit.UI.Controls">
+    <Grid.ColumnDefinitions>
+        <ColumnDefinition Width="*"/>
+        <ColumnDefinition Width="300"/>
+    </Grid.ColumnDefinitions>
+    <esri:MapView x:Name="MyMapView"/>
+    <toolkit:Legend GeoView="{Binding ElementName=MyMapView}"
```
Comment (dotMorten) [inline docs/legend.md:37]: This should be `{x:Bind MyMapView}`

File: docs/measure-toolbar.md [hunk 2]
```diff
@@ -13,6 +13,33 @@ Measure distances, areas, and features in a `MapView`.
...
+
+```xml
+<Grid xmlns:esri="using:Esri.ArcGISRuntime.UI.Controls"
+      xmlns:toolkit="using:Esri.ArcGISRuntime.Toolkit.UI.Controls">
+    <Grid.RowDefinitions>
+        <RowDefinition Height="50" />
+        <RowDefinition Height="*" />
+    </Grid.RowDefinitions>
+    <toolkit:MeasureToolbar HorizontalAlignment="Center"
+                            MapView="{Binding ElementName=MyMapView}" />
```
Comment (dotMorten) [inline docs/measure-toolbar.md:26]: `{x:Bind MyMapView}`

File: docs/overview-map.md [hunk 3]
```diff
@@ -28,60 +28,33 @@ OverviewMap has the following bindable properties:
...
-        <overviewmap:OverviewMap
-            Margin="4"
-            GeoView="{Binding ElementName=MySceneView}" />
-    </Grid>
-</UserControl>
+<Grid xmlns:esri="using:Esri.ArcGISRuntime.UI.Controls"
+      xmlns:toolkit="using:Esri.ArcGISRuntime.Toolkit.UI.Controls">
+    <esri:MapView x:Name="MyMapView" />
+    <toolkit:OverviewMap Margin="4"
+                         GeoView="{Binding ElementName=MyMapView}" />
```
Comment (dotMorten) [inline docs/overview-map.md:48]: `{x:Bind MyMapView}`

**Review by dotMorten — APPROVED** [review]

**Pull request merged**